autotest/ogr/ogr_flatgeobuf.py

```diff
@@ -410,3 +415,109 @@ def test_ogr_flatgeobuf_datatypes():
     assert f['double'] == 1.25
     assert f['string'] == 'my string'
     assert f['datetime'] == '2019/10/15 12:34:56.789+00'
+
+
+###############################################################################
+do_log = False
+
+class WFSHTTPHandler(BaseHTTPRequestHandler):
+
+    def log_request(self, code='-', size='-'):
+        pass
+
+    def do_GET(self):
+
+        try:
+            if do_log:
+                f = open('/tmp/log.txt', 'a')
+                f.write('GET %s\n' % self.path)
+                f.close()
+
+            if self.path.find('/fakewfs') != -1:
+
+                if self.path == '/fakewfs?SERVICE=WFS&REQUEST=GetCapabilities' or \
+                        self.path == '/fakewfs?SERVICE=WFS&REQUEST=GetCapabilities&ACCEPTVERSIONS=1.1.0,1.0.0':
+                    self.send_response(200)
+                    self.send_header('Content-type', 'application/xml')
+                    self.end_headers()
+                    f = open('data/testfgb/wfs/get_capabilities.xml', 'rb')
+                    content = f.read()
+                    f.close()
+                    self.wfile.write(content)
+                    return
+
+                if self.path == '/fakewfs?SERVICE=WFS&VERSION=1.1.0&REQUEST=DescribeFeatureType&TYPENAME=topp:states':
+                    self.send_response(200)
+                    self.send_header('Content-type', 'application/xml')
+                    self.end_headers()
+                    f = open('data/testfgb/wfs/describe_feature_type.xml', 'rb')
+                    content = f.read()
+                    f.close()
+                    self.wfile.write(content)
+                    return
+
+                if self.path == '/fakewfs?SERVICE=WFS&VERSION=1.1.0&REQUEST=GetFeature&TYPENAME=topp:states':
+                    self.send_response(200)
+                    self.send_header('Content-type', 'application/xml')
+                    self.end_headers()
+                    f = open('data/testfgb/wfs/get_feature.xml', 'rb')
+                    content = f.read()
+                    f.close()
+                    self.wfile.write(content)
+                    return
+
+                if self.path == '/fakewfs?OUTPUTFORMAT=application/flatgeobuf&SERVICE=WFS&VERSION=1.1.0&REQUEST=GetFeature&TYPENAME=topp:states':
+                    self.send_response(200)
+                    self.send_header('Content-type', 'application/flatgeobuf')
+                    self.end_headers()
+                    f = open('data/testfgb/wfs/get_feature.fgb', 'rb')
```

Comment:
any chance of using a smaller test file with less features & attributes ?

Reply:
Good idea, I've chosen another smaller layer from the geoserver demo test data.